core-web/libs/sdk/client/src/lib/client/sdk-js-client.ts

```diff
@@ -319,16 +321,24 @@ export class DotCmsClient {
 
             // Format the URL correctly depending on the 'path' value
             const formattedPath = path === '/' ? '/' : `/${path}`;
-            const url = `${this.config.dotcmsUrl}/api/v1/nav${formattedPath}${
+            const url = `${this.#config.dotcmsUrl}/api/v1/nav${formattedPath}${
                 queryParams ? `?${queryParams}` : ''
             }`;
 
-            const response = await fetch(url, this.requestOptions);
+            const response = await fetch(url, this.#requestOptions);
 
             return response.json();
         }
     };
 
+    static init(config: ClientConfig): DotCmsClient {
+        return (this.instance = new DotCmsClient(config));
+    }
```

Comment:
@fmontes @KevinDavilaDotCMS What do you think? I don't see this happening, but if the users send another config, what should we do? 

We already have the `instance` properties that allow us to take the config we need in our `SDK` so I don't see the need to deprive the user of not rewriting the configuration.

Reply:
Remember that the instance is static, this means that any `new` call will override the config and therefore we will have the last config available instead of 2 different instances. I think we should return the instance if it does exist to avoid unpredictable behaviors.